src/lib/remark-plugins/remark-plugin-adjust-link-urls/helpers.ts

```diff
@@ -12,6 +10,23 @@ const getIsInternalPath = (url: string) => {
 	}
 }
 
+/**
+ * Handles processing a Link or Definition node in documentation content. Returns
+ * the adjusted node URL.
+ */
+const processDocsLinkNode = ({
+	node,
+	currentPath,
+	urlAdjustFn,
+}: {
+	node: Link | Definition
+	currentPath: string
+	urlAdjustFn: (url: string) => string
+}) => {
+	const urlToAdjust = preAdjustUrl({ currentPath, url: node.url })
+	return urlAdjustFn(urlToAdjust)
```

Comment:
🗒️ This is the old body of `remarkPluginAdjustLinkUrls`, only adjusted to return the result of `urlAdjustFn` rather than set the `node.url` variable to the result. The plugin handles setting the `node.url` variable to the result of this new function.